cloudinit/distros/debian.py

```diff
@@ -282,7 +296,7 @@ def update_locale_conf(locale, sys_path, keyname="LANG"):
     )
 
 
-def regenerate_locale(locale, sys_path, keyname="LANG"):
+def regenerate_locale(locale, sys_path, keyname="LANG", install_callback=None):
```

Comment:
Why are we passing an install function here rather than calling `self.install_packages` in the body?

Nit: I also wouldn't call this a callback

Reply:
@TheRealFalcon do you mean do the subp.which call  and install_packages up in `Distro.apply_locale`?The regenerate_locale and update_locale_conf are functions which aren't defined on the class. Rather than pulling that function up into the class as a method, I chose to pass the Distro.install_packages function into those helpers. If you think it makes more sense (because we are using self.install_packages) to pull regenerate_locale and update_locale_conf up into the debian.Distro class I can do that instead.